examples/README.md

```diff
@@ -29,3 +29,13 @@ This ASP.NET web application is a .NET Framework MVC 5 app. It leverages the [co
 
 ## [WebFormApp (.NET Framework)](./DotNetFramework/WebFormApp)
 This ASP.NET web application is a .NET Framework Web Forms app. It demonstrates how to leverage the App Configuration [.NET Standard provider library](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration/) to achieve dynamic configuration and control feature launches with feature flags. The same technique applies to .NET Framework MVC apps.
+
+## [Spring Samples](https://github.com/Azure-Samples/azure-spring-boot-samples/)
```

Comment:
Link till AppConfiguration folder

Reply:
It would cause the link need to be updated every release https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_v4.2.0/appconfiguration

Reply:
Can we point to the main branch?

Reply:
The “main” branch isn’t the main branch. So, no.

Reply:
Can we create a tag pointing to the latest examples, or push the latest examples to the main? Any other ideas we can avoid the version being part of the URI?

Reply:
This is owned by the Spring team, and they are putting everything into verion branches. I can follow up with them about it. They are just making the latest branch the default one, which doesn't work for linking.

Reply:
Yes, please. Alternatively, you can create an aka.ms link and update it whenever you have a newer version.